Add bt support/external adapter

pupdum opened the pull request:
This enables the Bluez Bluetooth stack and hotplug support for USB Bluetooth hardware to the OT2 configuration. It also newly adds the [Bluepy](https://github.com/IanHarvey/bluepy) package to Buildroot and the OT2 defconfig.

This will extend the capabilities of the OT2 and allow for more communication options. 

These changes were implemented by adding the new package, enabling Bluez tools and utils packages, and adding new systemd units and a udev rule. I tried to make the rule generic enough that it could match with most if not all USB Bluetooth dongles. 

As far as testing, I built these changes and have used a Bluetooth Low Energy capable dongle on an OT2 to send and receive data. I have successfully tested the plug and play capabilities with one device before the udev rule was relocated to its own file. I currently do not have access to an OT2 so someone else might want to check if that change broke anything.

Commit:
package/python-bluepy: new package ver. 1.3.0

Package added because it cannot be pip installed from a buildroot system
Requires libglib2.0-dev (BR2_PACKAGE_LIBGLIB2 ?) to compile.

Signed-off-by: August Koehler <august.koehler@opentrons.com>

Commit:
Select package python-bluepy

Commit:
Select bluez tools and utils packages

Adds the bluetooth stack and related utilities.

Commit:
Add services and udev rules for USB BT hardware

New units for systemd to enable a USB bluetooth adapter on startup
or upon hotplugging.

New udev rule to trigger the btusbstarter@ service upon detection of
any USB bluetooth adapter.

pupdum committed:
Update ot2_defconfig

Removed an extraneous line selecting an unused package.

sfoster1 commented on configs/ot2_defconfig:
```diff
@@ -43,12 +43,12 @@ BR2_PACKAGE_RPI_USERLAND=y
 BR2_PACKAGE_UBOOT_TOOLS=y
 BR2_PACKAGE_USBMOUNT=y
 BR2_PACKAGE_PYTHON3=y
-BR2_PACKAGE_PYTHON3_BZIP2=y
```
We want this still I think... savedefconfig keeps deleting it for some reason but for now let's manually re-add it

pupdum replied:
Ahh, yeah there do seem to be some unexpected changes. I'll fix that.

sfoster1 reviewed (changes requested):
One little change in the defconfig, but otherwise looks good!

pupdum committed:
Select package python3 bzip2 

This package was unintentionally removed from the defconfig so it is added again.

pupdum commented:
Branch needs to be renamed.
